### PR TITLE
Updating author and contributor lists

### PR DESCRIPTION
**Pull request by nickerso**
I believe this accurately captures people's contributions to the CellML 2.0 specification. (if I get the PR correct this time!) will close cellml/cellml-specification#91

### CODE REVIEW

**Comment by nickerso** [comment]
I have all the names, I'm pretty sure, but need to check the classification :)

File: reference/sectionA_definitions.inc [hunk 1]
```diff
@@ -10,22 +10,29 @@ Jonathan Cooper
...
+David Brooks
+Edmund J. Crampin
+Jesús Carro Fernández
+Peter J. Hunter
+Gary R. Mirams
+Maxwell L. Neal
 
 The authors also wish to acknowledge the significant contribution of the (discontinued) draft CellML 1.2 specification, much of the text of which was incorporated into this specification - although the semantics have changed considerably.
 The CellML 1.2 specification was itself the result of a collaborative effort by a number of researchers during 2008-2011\:
 Andrew K. Miller (who wrote the text reused here),
```
Comment (agarny) [inline reference/sectionA_definitions.inc]: Alphabetical order or do we want to make an exception for Andrew?

Comment (nickerso) [reply reference/sectionA_definitions.inc]: yep, that was the one exception to alphabetical order.

File: reference/sectionA_definitions.inc [hunk 2]
```diff
@@ -10,22 +10,29 @@ Jonathan Cooper
...
-Justin Marsh,
 Alan Garny,
-Randall Britten,
-Poul Nielsen
-and David P. Nickerson
+Peter J. Hunter
+Justin Marsh,
+Poul Nielsen,
+David P. Nickerson,
+and Hugh Sorby
```
Comment (agarny) [inline reference/sectionA_definitions.inc]: Was Hugh involved in CellML 1.2? (Genuinely wondering.)

File: reference/sectionA_definitions.inc (hunk 1 above)
Comment (agarny) [reply reference/sectionA_definitions.inc]: I am happy with that. Just wanted to be sure it wasn't an overlook.

**Pull request merged**